packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -2597,6 +2597,165 @@ module.exports = function (config, DB) {
       })
     })
 
+    describe('account subscriptions', () => {
+      const subscriptionIds = [...Array(25)].map(() => 'sub-' + Math.random())
+      let account
+
+      beforeEach(async () => {
+        account = createAccount()
+        return db.createAccount(account.uid, account)
+      })
+
+      it('should create a new subscription for account', async () => {
+        const result =
+          await db.createAccountSubscription(account.uid, subscriptionIds[0], 'prod0', Date.now())
+        assert.deepEqual(result, {})
+      })
+
+      it('should fail to create for unknown account', async () => {
+        try {
+          await db.createAccountSubscription('12312312312', subscriptionIds[1], 'prod2', Date.now())
+          assert.fail()
+        } catch (err) {
+          assert.equal(err.errno, 116, 'not found')
+        }
+      })
+
+      it('should fail to create duplicate subscription id on the same account', async () => {
+        try {
+          const account2 = createAccount()
+          await db.createAccount(account2.uid, account2)
```

Comment:
Tiny nit, but does this test need to create `account2`? Looks like it only uses `account` unless I'm missing something.

Reply:
Oh good catch, that was a copypasta good